eks/multicam_smoother.py

```diff
@@ -27,7 +28,8 @@ def fit_eks_mirrored_multicam(
     quantile_keep_pca: float = 95,
     avg_mode: str = 'median',
     var_mode: str = 'confidence_weighted_var',
-    verbose: bool = False
+    verbose: bool = False,
+    maha: bool = False
```

Comment:
change from `maha` to `inflate_vars` to make it a bit more clear what this is doing